src/extension-test/unit/cursor-doc/token-cursor-test.ts

```diff
@@ -1,458 +1,457 @@
 import * as expect from 'expect';
 import { LispTokenCursor } from '../../../cursor-doc/token-cursor';
-import { currentEnclosingFormToCursor } from '../../../util/cursor-get-text';
 import { docFromTextNotation, textAndSelection } from '../common/text-notation';
 
 describe('Token Cursor', () => {
   describe('backwardWhitespace', () => {
     it('it moves past whitespace', () => {
       const a = docFromTextNotation('a •|c');
       const b = docFromTextNotation('a| •c');
-      const cursor: LispTokenCursor = a.getTokenCursor(a.selectionLeft);
+      const cursor: LispTokenCursor = a.getTokenCursor(a.selection.anchor);
```

Comment:
Any particular reason you choose `anchor` over `active` for these? It doesn't matter much, since they are the same, but to me the thing that blinks is the `active` thing. 😄

Reply:
Yeah I agree - good point.

In general most of the time `left` is anchor in the codebase (before this PR I suppose) in selections, and my having used anchor in this case is just an oversight/accident